lib/connection.ts

```
@@ -1,13 +1,46 @@
+    static initialize_(endpoint: string, project: string, apiKey: string) {
+
+        if (this.client) {
+            return; // Prevent re-initialization
+        }
+        this.client = new Client();
+        this.client.setEndpoint(endpoint).setProject(project);
+    }
+
+    static initialize(endpoint: string, project: string) {
+
+        if (this.client) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)